pyproject.toml

```
@@ -14,10 +14,10 @@ requires-python = ">=3.8"
 dependencies = [
   # intentionally loose. perhaps these should be vendored to not collide with user code?
   "attrs>=20.1,<24",
-  "fastapi>=0.75.2,<0.99.0",
+  "fastapi>=0.75.2",
```

Bump minimum requirement to first version of FastAPI supporting Pydantic 2

```suggestion
  "fastapi>=0.100.0",
```